pypika/queries.py

```diff
@@ -374,11 +397,18 @@ def __init__(self, quote_char='"', dialect=None, wrap_union_queries=True, wrappe
         self._groupbys = []
         self._with_totals = False
         self._havings = None
+
         self._orderbys = []
+        self._ordertype = None
+
         self._joins = []
         self._unions = []
 
         self._limit = None
+
+        self._limit_min = None
```

Comment:
instead of adding these as extra fields, I think  it would be reasonable to recycle the existing limit function and pass a tuple as the argument to it. Wdyt?

Reply:
yes.
I added this feature so as not to mess up the code.